tests/BaseBuilderTest.php

```diff
@@ -5,29 +5,61 @@
 namespace Doctrine\Tests\RST;
 
 use Doctrine\RST\Builder;
+use Doctrine\RST\Configuration;
+use Doctrine\RST\ErrorManager;
+use Doctrine\RST\ErrorManager\ErrorManagerFactory;
 use Exception;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
 use function file_get_contents;
 use function shell_exec;
 
 abstract class BaseBuilderTest extends TestCase
 {
-    /** @var Builder */
-    protected $builder;
+    protected Configuration $configuration;
+    protected Builder $builder;
+    /** @var ErrorManager|MockObject */
+    protected $errorManger;
```

Comment:
:yum: https://dict.leo.org/franz%C3%B6sisch-deutsch/manger

```suggestion
    protected $errorManager;
```